api/lanes.py

```diff
@@ -107,23 +107,24 @@ def create_default_lanes(_db, library):
 
     The database will have the following top-level lanes for
     each large-collection:
-    'Adult Fiction', 'Adult Nonfiction', 'Young Adult Fiction',
-    'Young Adult Nonfiction', and 'Children'.
+    'Adult Fiction', 'Adult Nonfiction', 'Young Adult',
+    and 'Children'.
```

Comment:
I think the old version of this comment is correct, don't see any changes related to this.

Reply:
You're right, this is a proposed change I decided not to make (yet?)